main/src/ca/uwaterloo/flix/language/ast/package.scala

```diff
@@ -144,6 +144,7 @@ package object ast {
                   verifier: Long,
                   lambdaLift: Long,
                   tailrec: Long,
+                  inliner: Long,
```

Comment:
You also need to make some additional changes to ensure that the phase shows up in the online debugger.

Check out how the time object is used (see e.g. in RestServer).

Then try to run with `--monitor` and verify that this phase takes a reasonable amount of time.